package.json

```diff
@@ -52,8 +52,8 @@
   },
   "scripts": {
     "start": "node server/index.js",
-    "build:main": "node-sass --include-path custom/styles --include-path styles/partials --source-map public/css/style.css.map --source-map-contents styles/style.scss public/css/style.css",
-    "build:inline": "node-sass --include-path custom/styles --include-path styles/partials styles/errors.scss public/css/errors.css",
+    "build:main": "sass --load-path custom/styles --load-path styles/partials --source-map styles/style.scss public/css/style.css",
```

Comment:
It seems that the source maps are generated automatically by dart sass: https://sass-lang.com/documentation/cli/dart-sass

The --source-map does not seem to exist anymore